ambry-named-mysql/src/main/java/com/github/ambry/named/NamedBlobState.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2022 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.named;
+
+public enum NamedBlobState {
+  WIP(0),
```

Comment:
We should use InProgress now we are going to use integral value.

Reply:
Done in [7403e50](https://github.com/linkedin/ambry/pull/2143/commits/7403e5066fef239a52857ab4f2cf4d8fb1b137d3)
Can you take another look?